tests/scenario_tests/test_block_suggestion.py

```diff
@@ -180,6 +180,34 @@ def test_failure_multi(self):
         assert response.status == 404
         assert_auth_test_count(self, 1)
 
+    def test_empty_options(self):
+        app = App(
+            client=self.web_client,
+            signing_secret=self.signing_secret,
+        )
+        app.options("mes_a")(show_empty_options)
+
+        request = self.build_valid_multi_request()
+        response = app.dispatch(request)
+        assert response.status == 200
+        assert response.body == """{"options": []}"""
```

Comment:
Due to the bug, this response body used to be empty string `""`. That's why the select menu options are not updated in the reported scenario.